datafusion/core/tests/data/partitioned_table/c_date=2018-11-13/timestamps.csv

```diff
@@ -0,0 +1,2 @@
+Jorge,2018-12-13T12:12:10.011Z
```

Comment:
These files used to be generated programatically in Rust, instead I have checked them in for the tests